test/feature/features/exit.feature

```diff
@@ -0,0 +1,7 @@
+Feature: Exit
+  I should logout of AutolabJS when I run exit command
+
+  Scenario: Exit command
+    Given I have already logged in
+    When I run exit command
+    Then My login credentials should be removed
```

Comment:
other useful scenarios are:
1. execution of `autolabjs exit` command twice in correct location containing AutolabJS assignment.
1. execution of `autolabjs exit` in any random location not containing AutolabJS assignment.